forms-flow-bpm/Dockerfile

```diff
@@ -52,4 +51,4 @@ RUN mvn -s /usr/share/maven/ref/settings-docker.xml exec:java -e -Dmain.class=co
 
 EXPOSE 8080
 
-ENTRYPOINT ["/bin/bash", "-c", "lsyncd lsyncd.conf; mvn -s /usr/share/maven/ref/settings-docker.xml package; java -Djava.security.egd=file:/dev/./urandom -jar /tmp/target/forms-flow-bpm.jar"]
\ No newline at end of file
+ENTRYPOINT ["/bin/bash", "-c", "lsyncd lsyncd.conf; mvn -s /usr/share/maven/ref/settings-docker.xml -Dmaven.test.skip=true package; java -Djava.security.egd=file:/dev/./urandom -jar /tmp/target/forms-flow-bpm.jar"]
```

Comment:
Just added in this so we don't have to run the tests every time we make a change to the bpm